packages/components/src/components/editor.tsx

```diff
@@ -29,22 +38,23 @@ export const Editor = () => {
   const [responseValue, setResponseValue] = useAtom(responseValueAtom)
   const [trpcClient] = useAtom(trpcClientAtom)
   const [config] = useAtom(configAtom)
-  const evalFunction = useMemo(() => config.request.batching ? batchEval : serialEval, [config.request.batching])
+  const evalFunction = useMemo(() => config?.request.batching ? batchEval : serialEval, [config.request.batching])
 
   const refreshTypes = useCallback(async () => {
     if (!editorView) return
 
     try {
-      const types = await makePlaygroundRequest('getTypes', {
+      const response = await makePlaygroundRequest('getTypes', {
         playgroundEndpoint: config.playgroundEndpoint,
       })
+      setTypes(response)
 
       editorView.dispatch(injectTypes({
-        '/index.d.ts': types.join('\n'),
+        '/index.d.ts': response.tsTypes.join('\n'),
```

Comment:
Maybe we should change the name of the operation from `getTypes`? Maybe to `getRouterSchema` instead.

Reply:
done